packages/shared-state-network_nodes/files/etc/uci-defaults/shared-state-network_nodes-cron

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+unique_append()
```

Comment:
this is not the first time I see this function.
should it become part of a library?

Reply:
Yes, when we create the lime-utils package :)